tests/by-util/test_cksum.rs

```diff
@@ -602,7 +602,6 @@ fn test_reset_binary() {
         .stdout_contains("d41d8cd98f00b204e9800998ecf8427e  ");
 }
 
-#[ignore = "issue #6375"]
 #[test]
```

Comment:
I guess the comment at the end of the function (line 621) is no longer needed?

Reply:
It is not indeed, thanks for catching this !

Reply:
done :+1: